users/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
```

Comment:
suggestion: you should add a new serializer for user login and keep the current serializer for user registration.